cmake/multi_image.cmake

```diff
@@ -335,6 +335,8 @@ function(add_child_image_from_source)
       # Place the result in the CMake cache and remove local scoped variable.
       foreach(file CONF_FILE DTC_OVERLAY_FILE)
         if(DEFINED ${ACI_NAME}_${file})
+          zephyr_file_suffix(${ACI_NAME}_${file} SUFFIX ${${ACI_NAME}_FILE_SUFFIX})
```

Comment:
to be addressed, or an acceptable reason for why to accept this is given.

See also: https://github.com/gmarull/sdk-nrf/pull/13#discussion_r1570631752